Dockerfile.dapper

```diff
@@ -52,6 +52,7 @@ RUN cd /usr/src/libqcow-20181117 && \
 RUN cd /usr/src/libqcow-20181117 && \
     make -j$(nproc) && \
     make install
+RUN ldconfig
```

Comment:
Not sure why this is now necessary?
Since it worked previously, but I saw the thread on slack

Reply:
Previously we didn't load the libqcow library in the unit test but now we do import the `pkg/backingfile` in the replica unit test [here](https://github.com/longhorn/longhorn-engine/blob/fd02e4955e965a1d8e1c0d1155b25816e5d3ce18/pkg/replica/backup_test.go#L8). This in turn import the `libqcow` package which inturn includes the `#include <libqcow.h>`. So, I guess we need to run this command so that unit test can find the header `libqcow.h`